changelog.md

```diff
@@ -1,5 +1,111 @@
+  * [macOS] Added support for (some) Hat/POV axis ([#1248](https://github.com/SFML/SFML/pull/1248))
+  * [Linux] Fixed `sf::Window::getPosition()` returning incorrect position because of differences in window managers ([#1228](https://github.com/SFML/SFML/pull/1228), [#1266](https://github.com/SFML/SFML/pull/1266))
+  * [Linux] Fix X11 key repeat handling not filtering out events from other windows ([#1291](https://github.com/SFML/SFML/pull/1291), [#1223](https://github.com/SFML/SFML/pull/1223), [#1230](https://github.com/SFML/SFML/pull/1230))
+  * [Linux] Restore fullscreen of a non-visible window ([#1339](https://github.com/SFML/SFML/pull/1339))
+  * [iOS] Fix build error when using @import ([#1272](https://github.com/SFML/SFML/pull/1272))
+  * [iOS] Don't need to find vorbisfile or vorbisenc ([#1347](https://github.com/SFML/SFML/pull/1347))
+  * [Windows] Prevent uninitialized read by zeroing memory ([#1264](https://github.com/SFML/SFML/pull/1264))
+  * [Windows] Fixed modifier keys handling ([#1357](https://github.com/SFML/SFML/pull/1357))
+  * [Android] Return correct key code for delete/backspace ([#1362](https://github.com/SFML/SFML/pull/1362))
+
+### Graphics
+
+**Features**
+
+  * Implemented additional line spacing and letter spacing in `sf::Text` ([#928](https://github.com/SFML/SFML/pull/928), [#1366](https://github.com/SFML/SFML/pull/1366))
+  * Added `sf::VertexBuffer` class ([#1308](https://github.com/SFML/SFML/pull/1308))
+  * Added GPU local texture copying support, allowing performance optimizations and texture swapping ([#1119](https://github.com/SFML/SFML/pull/1119), [#1320](https://github.com/SFML/SFML/pull/1320))
+  * Optimize performance by skipping `glTexCoordPointer()` call if not needed ([#1015](https://github.com/SFML/SFML/pull/1015))
+  * Generate shape outline vertices only if necessary ([#1356](https://github.com/SFML/SFML/pull/1356))
+  * Removed dependency to libjpeg, stb_image_write now supports writing JPEG files ([#1279](https://github.com/SFML/SFML/pull/1279))
+  * Enable comparing `sf::Transform` and optimize resetting OpenGL back to the identity matrix ([#1298](https://github.com/SFML/SFML/pull/1298))
+
+**Bugfixes**
+
+  * Fixed typo in assert in `sf::Texture::update()` ([#1210](https://github.com/SFML/SFML/pull/1210))
+  * Properly free memory in `sf::Font::cleanup()` ([#1119](https://github.com/SFML/SFML/pull/1119))
+  * Fixed memory leak in `sf::Font` ([#1216](https://github.com/SFML/SFML/pull/1216))
+  * Fix OpenGL texture coordinate pointer not being updated correctly under certain conditions ([#1297](https://github.com/SFML/SFML/pull/1297))
+  * Fix for broken text when the font is reloaded ([#1345](https://github.com/SFML/SFML/pull/1345))
+  * Fix memory leak in `sf::Text` ([#1360](https://github.com/SFML/SFML/pull/1360))
+  * Refactoring of `sf::Text` ([#928](https://github.com/SFML/SFML/pull/928))
```

Comment:
This description and PR don't match. Also already mentioned above. Picked the wrong number?